### ISSUE TITLE
Recommendations on half body training and inference

### ISSUE THREAD

**Issue by nfrik** [NVIDIA/vid2vid#77]
I was able to obtain fairly low generator loss on the pose to body model trained for 10 epochs on a ~2 minute video (see below) of the upper body at 256 resolution with densepose and openpose. However inference on the similar type of upper body data, returns extremely low quality, blurry images even when remove_face_labels flag is set, as stated in the documentation. I wonder if there are any specific recommendations on how half-body videos need to be treated and which hyperparameters would be responsible for increasing inference accuracy besides remove_face_labels flag? Thank you.

Video used:
[https://www.youtube.com/watch?v=c8EJNVC_MxA]

Inference images:
![iterm2 vewtqc fake_b_frame_0030_orig](https://user-images.githubusercontent.com/1614059/49090985-e63c6280-f22c-11e8-855b-d5802dd5c7c4.jpg)
